scipy/stats/_binned_statistic.py

```diff
@@ -310,10 +309,10 @@ def binned_statistic_2d(x, y, values, statistic='mean',
     >>> y = [2.1, 2.6, 2.1, 2.1]
     >>> binx = [0.0, 0.5, 1.0]
     >>> biny = [2.0, 2.5, 3.0]
-    >>> ret = stats.binned_statistic_2d(x, y, None, 'count', bins=[binx,biny])
+    >>> ret = stats.binned_statistic_2d(x, y, x, 'count', bins=[binx,biny])
```

Comment:
This change should not have been necessary, and should be undone when the issue noted in my comment is fixed.

Reply:
yes. I'll add a change to revert this one from `x` -> `None`